person.rb

```diff
@@ -0,0 +1,18 @@
+class Person
+  attr_accessor :id, :name, :age
+
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @id = Random.rand(1..100)
+    @age = age
+    @name = name
+    @parent_permission = parent_permission
+  end
+
+  def of_age?
+    @age >= 18
+  end
+
```

Comment:
- [ ] It is required that you make this a private method. Kindly do that.

Reply:
done, thanks!